### ISSUE TITLE
How to Know the compliance level of WCAG 2.1 with Axe-core report?

### ISSUE THREAD

**Issue by VishnuVarsan** [dequelabs/axe-core#4159]
### Product

axe-core

### Question

May I know, How to calculate the Compliance level of the WCAG report with the help of Axe-core report?

What are conditions or any measures provided to calculate conformance level with this report.?

**Comment by straker** [comment]
Thanks for the question. So the problem with trying to calculate a conformance report from axe-core is that axe-core's rules do not fully cover all the Success Criterion for the guidelines. So even though the `image-alt` rule can help with knowing if there is alternative text for an image, it cannot judge if that alternative text serves the equivalent purpose of the image.

**Comment by VishnuVarsan** [comment]
Thanks for your update!
